src/Mapbender/CoreBundle/Component/Element.php

```diff
@@ -368,10 +368,10 @@ public function trans($key, array $parameters = array(), $domain = null, $locale
      *************************************************************************/
 
     /**
-     * Get the element configuration form type.
+     * Get the element configuration form type. By default, this class needs to have the same name
+     * as the element, suffixed with "AdminType" and located in the Element\Type sub-namespace.
      *
-     * Override this method to provide a custom configuration form instead of
-     * the default YAML form.
+     * Override this method and return null to get a simple YAML entry form.
      *
```

Comment:
For the change in behavior redocumented here, see https://github.com/mapbender/mapbender/commit/387adf1e90e733249f436ec67ad11daf05bc4249#diff-aa19882f5b611faae7e41b317a61faf7L323
Could be another BC break.